src/style.css

```diff
@@ -0,0 +1,107 @@
+* {
+  box-sizing: border-box;
+}
+
+.html {
+  font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  font-size: 100px;
+  color: white;
+  justify-content: center;
+  align-items: center;
+}
+
+.block {
+  height: 300px;
+  min-width: 300px;
+  flex: 0 0 33.3%;
+  text-align: center;
+  line-height: 300px;
+  font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;
```

Comment:
no need to repeat font-family. You inherit it by default

Reply:
@aionova does not inherit, i checked

Reply:
When i delete font-family from .block the style is not applied

Reply:
@aionova ooooooo thanks))